react/features/app/actions.js

```diff
@@ -14,11 +14,23 @@ import {
 } from './actionTypes';
 import {
     _getRoomAndDomainFromUrlString,
-    _getRouteToRender
+    _getRouteToRender,
+    init
 } from './functions';
 import './reducer';
 
 /**
+ * Temporary solution. Should dispatch actions related to
+ * initial settings of the app like setting log levels,
+ * reading the config parameters from query string etc.
+ *
+ * @returns {Function}
+ */
+export function appInit() {
```

Comment:
I think this "action" is pointless and I believe it should be removed:

1. It is dispatched by `App.web.js` on `componentWillMount`. However, `AbstractApp` dispatches `appWillMount` on `componentWillMount`. Whatever `App.web.js` does with `appInit` can already be done on `appWillMount`.

2. It's not really a Redux action that anyone else can intercept or reduce so calling it a Redux action is superfluous.